aspnetcore/tutorials/razor-pages/model.md

```diff
@@ -1,59 +1,491 @@
 ---
-title: Part 2, add a model to a Razor Pages app in ASP.NET Core
+title: Part 2, add a model
 author: rick-anderson
-description: Part 2 of tutorial series on Razor Pages.
+description: Part 2 of tutorial series on Razor Pages. In this section, model classes are added.
 ms.author: riande
-ms.date: 12/05/2019
-no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+ms.date: 09/30/2020
+no-loc: [Index, Create, Delete, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/razor-pages/model
 ---
 # Part 2, add a model to a Razor Pages app in ASP.NET Core
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-5.0"
+
+<!-- In the next update on the CLI version, let the scaffolder do the same work the VS driven scaffolder does. That is, create the DB context, etc -->
+
+In this section, classes are added for managing movies in a database. The app's model classes use [Entity Framework Core (EF Core)](/ef/core) to work with the database. EF Core is an object-relational mapper (O/RM) that simplifies data access. You write the model classes first, and EF Core creates the database.
+
+The model classes are known as POCO classes (from "**P**lain-**O**ld **C**LR **O**bjects") because they don't have any dependency on EF Core. They define the properties of the data that are stored in the database.
+
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie50) ([how to download](xref:index#how-to-download-a-sample)).
+
+## Add a data model
+
+# [Visual Studio](#tab/visual-studio)
+
+1. In **Solution Explorer**, right-click the *RazorPagesMovie* project > **Add** > **New Folder**. Name the folder *Models*.
+1. Right-click the *Models* folder. Select **Add** > **Class**. Name the class *Movie*.
+1. Add the following properties to the `Movie` class:
+
+   [!code-csharp[](~/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie22/Models/Movie.cs?name=snippet1)]
+
+The `Movie` class contains:
```

Comment:
All this is repeated by the 3 tabs. Move outside tabs.

Reply:
@Rick-Anderson, thanks. When I looked at this repeated text early on in the draft, I was reluctant to move it outside the three tabs since it is not at the beginning or end for all three IDE's. for that small amount of repeated text. It is properly imbedded in the middle of the VS Code specific instruction.  

To eliminate the duplicated text, it looked like I would have to break up the tabbed areas, making the tabs a bit more busy.  So I figured that was why you left it that way before in your original version. 

This is the current organization:

![image](https://user-images.githubusercontent.com/10985336/98293585-7203b100-1f63-11eb-90cd-be4e2a765b5a.png)

I will move it then to this organization, which adds tabs, unless you were thinking of a different organization:

![image](https://user-images.githubusercontent.com/10985336/98293273-e427c600-1f62-11eb-8c3a-d2f2fb93bee9.png)

Reply:
I don't think I can even have a single VS Code tab like that to try to make this work.  So maybe you were thinking of some other organization?

Reply:
Hold off on this